### ISSUE TITLE
Skip link target does not get focus (critical)

### ISSUE THREAD

**Issue by MelanieHotchkiss** [magda-io/magda#1906]
### Problem description
Accessibility issue no 3 - the skip to navigation link skips to the navigation menu but does not give focus to the first menu item.

### Problem reproduction steps
Figure out if design system and who needs to do it (DTA or Data61)

### Screenshot / Design / File reference
Intopia suggestion:
[3] Skip link target does not get focus
Type: WCAG 2.1 failure
Fails 2.4.3 Focus Order - Level A
Location: Skip to Navigation link, header, all pages
Platform: Desktop
Severity: Critical
What’s the issue?
The Skip to Navigation link skips to the navigation menu but does not give focus to the first menu item.
Why it matters
Screen reader and keyboard only users expect a Skip to Navigation link to take them to – and give focus to – the first item of the navigation menu.
Our recommendation
Given that the Skip to Navigation link in fact skips only one item – the logo – consideration should be given to removing the skip link altogether.
If it is determined preferable to retain the Skip to Navigation link, it can be made to work properly by removing the aria-hidden=”true” attribute from the target div.
Code sample:
<div class="col-md-8 col-xs-12">
<div class="au-accordion__body au-accordion--closed menu" aria-hidden="true" >
<nav class="navigation header-nav" id="nav">
<ul class="au-link-list au-link-list--inline">
…
</ul>
</nav>
</div>
</div>
